Content.Shared/Stealth/Components/StealthComponent.cs

```diff
@@ -19,6 +19,12 @@ public sealed partial class StealthComponent : Component
     [DataField("enabled")]
     public bool Enabled = true;
 
+    /// <summary>
+    /// The creature will become visible at death.
```

Comment:
maybe it will be more correct with "/// The creature will continue invisible at death."

Reply:
done